tests/e2e/e2e_suite_test.go

```diff
@@ -190,11 +195,9 @@ func (s *Suite) TearDownSuite() {
 	s.T().Log("Starting tear down E2E test setup...")
 
 	if s.workloadPod != nil {
-		s.T().Logf("Starting delete of the workload pod: %s...", s.workloadPod.Name)
```

Comment:
The issue, with `s.T().Logf` that messages will appear only after test execution when golang test framework flushes the output buffer. It doesn't make sense to show the message after the fact that the workload pod was deleted.